client/elements/text/sc-simple-text.html

```diff
@@ -781,6 +830,51 @@ <h3>Could not find the sutta text.</h3>
                   }, 0);
               }
           }
+
+          _markupVarNote(noteData,referenceText) {
+              const noteItems = noteData.split(/ \| /);
+              var noteText = referenceText + '<paper-tooltip class="varnote"><table><tbody>';
+              var out = "";
```

Comment:
use `let` or `const` instead of `var`
Also please use single quotes `'` instead of double `"`